lib/lita/handlers/artifactory.rb

```diff
@@ -126,6 +126,11 @@ def promote(response)
           EOH
         end
 
+        # This event allows other lita utilities to act when projects are promoted.
+        robot.trigger(:artifactory_artifact_promoted, product: project, version: version)
```

Comment:
@tduffield Yep. Go with what you think is the best fix....was just sharing `def source_channel` as an example.